tensorflow/compiler/xla/service/gpu/gemm_thunk.cc

```diff
@@ -173,25 +186,128 @@ static Status DoGemm(int64_t batch_size, const MatrixDescriptor &lhs,
         /*leading dim of output=*/output_matrix.num_rows, output_matrix.stride,
         batch_size);
   }
-  return stream->ThenBlasGemm(
-      lhs.transpose, rhs.transpose, output_matrix.num_rows,
-      output_matrix.num_cols, /*size of reduce dim=*/lhs.reduced_dim(),
-      /*alpha=*/alpha, lhs.cast<Input>(),
-      /*leading dim of LHS=*/lhs.num_rows, rhs.cast<Input>(),
-      /*leading dim of RHS=*/rhs.num_rows,
-      /*beta=*/beta, &output_data,
-      /*leading dim of output=*/output_matrix.num_rows);
+  return stream->ThenBlasGemm(lhs.transpose, rhs.transpose,
+                              output_matrix.num_rows, output_matrix.num_cols,
+                              /*size of reduce dim=*/lhs.reduced_dim(),
+                              /*alpha=*/alpha, lhs.cast<Input>(),
+                              /*leading dim of LHS=*/lhs.num_rows,
+                              rhs.cast<Input>(),
+                              /*leading dim of RHS=*/rhs.num_rows,
+                              /*beta=*/beta, &output_data,
+                              /*leading dim of output=*/output_matrix.num_rows);
 }
 
-Status RunGemm(const GpuGemmConfig &gemm_config,
-               se::DeviceMemoryBase lhs_buffer, se::DeviceMemoryBase rhs_buffer,
-               se::DeviceMemoryBase output_buffer, se::Stream *stream,
-               bool implements_whole_instruction,
-               absl::optional<int64_t> profile_index,
-               se::blas::ProfileResult *profile_result,
-               absl::optional<se::blas::AlgorithmType> algorithm) {
-  VLOG(2) << "Executing a GemmThunk";
+#if GOOGLE_CUDA && CUDA_VERSION >= 11000
```

Comment:
Same comment for here: maybe we could think what would be a nicer runtime check?